src/openforms/js/components/admin/form_design/variables/registration/RegistrationsSummary.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import FAIcon from 'components/admin/FAIcon';
+import {BACKEND_OPTIONS_FORMS} from 'components/admin/form_design/registrations';
+import ButtonContainer from 'components/admin/forms/ButtonContainer';
+
+/**
+ *
+ * @param {Object} p
+ * @param {string} p.name - The name of the current registration backend
+ * @param {JSX.Element} p.registrationSummary - The rendered summary of the registration backend for a specific variable
+ * @returns {JSX.Element} - The rendered summary, containing the backend name, summary and edit icon
+ */
+const RegistrationSummary = ({name, registrationSummary}) => {
+  const intl = useIntl();
+
+  return (
+    <>
+      <div style={{display: 'flex', justifyContent: 'space-between'}}>
+        <div style={{overflow: 'hidden', whiteSpace: 'nowrap', textOverflow: 'ellipsis'}}>
+          <span>{name}</span>
+        </div>
+        <span>
+          <FAIcon
+            icon="edit"
+            title={intl.formatMessage({
+              defaultMessage: 'Edit variable registration',
+              description: "'Edit variable registration' icon label",
+            })}
+            extraClassname="fa-lg actions__action"
```

Comment:
allowing this for now until the styles are sorted out, but ideally the icons would follow the rest of the UI and be to the left:

```jsx
<div style={display: 'flex'}>
  <div className="actions actions--vertical">
    <FAIcon extraClassName="actions__action" />
  </div>
  <div>
    ...
  </div>
<div>
```